src/v2/Apps/Search/Components/NavigationTabs.tsx

```diff
@@ -17,17 +20,55 @@ export interface Props extends SystemContextProps {
 const MORE_TABS = ["tag", "city", "feature", "page"]
 
 const TAB_NAME_MAP = {
-  PartnerGallery: "Galleries",
-  PartnerInstitution: "Institutions",
-  PartnerInstitutionalSeller: "Institutions",
-  article: "Articles",
   artist: "Artists",
+  article: "Articles",
+  sale: "Auctions",
   artist_series: "Artist Series",
-  fair: "Fairs",
-  gene: "Categories",
   marketing_collection: "Collections",
+  fair: "Fairs",
   partner_show: "Shows",
-  sale: "Auctions",
+  PartnerInstitution: "Institutions",
+  PartnerInstitutionalSeller: "Institutions",
+  PartnerGallery: "Galleries",
+  gene: "Categories",
+}
+
+interface RouteTabProps extends BoxProps, RouterLinkProps {
+  text: string
+  count?: number
+}
+
+const RoundedRouteTab: React.FC<RouteTabProps> = ({ text, count, ...rest }) => {
+  const isActive = useIsRouteActive(rest.to, { exact: rest.exact ?? true })
+
+  return (
+    <RouteTab
+      height="auto"
+      borderWidth="1px"
+      borderStyle="solid"
+      borderRadius="20px / 50%"
+      borderColor={isActive ? "black60" : "black10"}
+      px={[1, 2]}
+      py={[0.5, 1]}
+      mr={1}
+      {...rest}
+    >
+      <Text variant="md" color={isActive ? "black100" : "black60"}>
+        {text}
+      </Text>
+      <Media greaterThan="xs">
+        {count && (
+          <Text
+            variant="xs"
+            color={isActive ? "blue100" : "black60"}
+            display="inline"
+          >
+            {count}
+          </Text>
+        )}
+      </Media>
+    </RouteTab>
```

Comment:
**suggestion**: please clarify with Jonathan whether there should be such a style with the hover effect?

https://user-images.githubusercontent.com/3513494/132865254-71e83241-e774-4b25-8f90-4fbd9aa7e174.mp4